flink-yarn-tests/src/test/java/org/apache/flink/yarn/YARNITCase.java

```diff
@@ -113,6 +113,8 @@ public void testPerJobMode() throws Exception {
 
 					assertThat(jobResult, is(notNullValue()));
 					assertThat(jobResult.getSerializedThrowable().isPresent(), is(false));
+
+					waitUntilApplicationFinished(applicationId, 10);
```

Comment:
Could you move `10` to a named private static final constant?

Reply:
Done